build/yamls/elk-flow-collector/logstash/logstash.conf

```diff
@@ -28,7 +28,7 @@ filter {
         }
     } else {
         mutate {
-            replace => { "[ipfix][bytes]" => "%{[ipfix][octetDeltaCount]}" }
+            replace => { "[ipfix][bytes]" => "%{[ipfix][reverseOctetDeltaCount]}" }
```

Comment:
I don't think so. The issue is related to packetDeltaCount rather than octetDeltaCount. It will be better if an issue is created to describe the problem in detail.

Reply:
@zyiou Not sure Piotr will open an issue. I think there is enough context there to fix the dashboard. It is good to improve the dashboard through feedback from Slack in addition to Issues on github repo. 

What I understood from the thread: there is an issue if there is only one flow record for the flow (short flows), then we do not show any data for those flows. 
Seems like ipfix.Packets and ipfix.Bytes are the delta counts. Could we look at changing to total packet counts

Reply:
@zyiou please feel free to open the issue yourself with the information from the Slack thread so we can track this

Reply:
@srikartati I read the thread again. It seems that the only remaining problem to be solved is changing ipfix.packets and ipfix.bytes to total packet and byte counts. Created an issue [here](https://github.com/vmware-tanzu/antrea/issues/1794). Let me know if I missed something. Thanks!